spec/requests/api/v1.0/portfolio_items_spec.rb

```diff
@@ -108,13 +108,16 @@
   describe 'POST /portfolio_items/{portfolio_item_id}/undelete' do
     let(:undelete) { post "#{api_version}/portfolio_items/#{portfolio_item_id}/undelete", :params => { :restore_key => restore_key }, :headers => default_headers }
     let(:restore_key) { Digest::SHA1.hexdigest(portfolio_item.discarded_at.to_s) }
+    subject { undelete }
```

Comment:
should we rename `undelete` to `restore`, to keep it in sync? Same thing for `describe` here

Reply:
Possibly, but that would be a breaking API change and would require a new version. Definitely outside the scope of this PR.

Reply:
Yes, I agree with you about API concern. My initial point is only in spec test, something like: 
```
let(:restore) { ... }
subject { restore }
...
it_behaves_like "action that tests authorization", :restore?,
```

Reply:
Oh, I see. Hmm, yeah we could do that. The endpoint still posts to `api/v1.x/portfolio_items/:id/undelete` though, so it's a little weird.

I'll change it though since we should definitely be trying to use the "restore" language wherever we can, I think.